pkg/pillar/dpcreconciler/linuxitems/bond.go

```diff
@@ -320,7 +323,10 @@ func (c *BondConfigurator) Delete(ctx context.Context, item depgraph.Item) error
 func (c *BondConfigurator) NeedsRecreate(oldItem, newItem depgraph.Item) (recreate bool) {
 	oldBondCfg := oldItem.(Bond)
 	newBondCfg := newItem.(Bond)
-	if !reflect.DeepEqual(oldBondCfg.BondConfig, newBondCfg.BondConfig) {
+	if oldBondCfg.LacpRate != newBondCfg.LacpRate ||
```

Comment:
Is this a separate bug fix or related to the VLAN change?

Reply:
This is just to optimize comparison and avoid using reflect.
I'm going to put this into a separate commit...

Reply:
Done